CHANGELOG.md

```diff
@@ -2,55 +2,56 @@
 
 ## Unreleased
```

Comment:
There a lot of changes in this file, it looks like a linter processed it.

Also, please add a new separate `Unreleased` section at the top, since this section corresponds to the 4.8 release which is already feature-frozen.

Reply:
may be done, please check

Reply:
The reason for these big changes was because Prettier rewrote some previously written data when saving the file.
Maybe you should add .prettierignore for CHANGELOG.md?